### DISCUSSION TITLE
Creating residue graph from PDB

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [a-r-j/graphein#369] [Q&A]
I want to generate protein residue graph from the PDB file which will eventually contain all of this properties in the residue graph
 
 Bond type                       Threshold
Covalent	                         —
Hydrophobic contacts	4
Hydrogen bond	3.5
Salt bridge	4
Cation–pi interaction	5
Perpendicular pi-stacking	5
Parallel pi-stacking	5

**Comment by a-r-j** [comment]
Have you consulted the documentation? There are some examples here: https://github.com/a-r-j/graphein/blob/master/notebooks/residue_graphs.ipynb

**Comment by shrimonmuke0202** [comment]
Actually, I am looking for Perpendicular pi-stacking and Parallel pi-stacking to construct the edge in the graph but I did not find any kind of function in the documentation.
